### PR TITLE
Updated Events Page

### LINKED ISSUE #70: Update Events Page
Add Tooling Series and Passion Talks to events page.
Add link to curriculum for each as well.

### PR DESCRIPTION
**Pull request by jodymlin**
Also added external links to curriculum, website, recordings, etc. 
Fixes #70

### CODE REVIEW

File: src/components/EventHighlight/EventHighlight.js [hunk 1]
```diff
@@ -93,6 +103,16 @@ function EventHighLight({ classes }) {
 				<Typography variant="body1">
 					{event.description}
 				</Typography>
+				<Link href={event.link}
+					target="_blank" rel="noopener noreferrer"
+					underline="none">
```
Comment (GalenWong) [inline src/components/EventHighlight/EventHighlight.js:108]:
```suggestion
				<Link 
				  href={event.link}
					target="_blank" 
					rel="noopener noreferrer"
					underline="none"
			  >
```
If you were to break up props into the multiple lines, keep only one prop on each line and keep the `>` on a line of its own!

File: src/components/EventHighlight/EventHighlight.js [hunk 2]
```diff
@@ -93,6 +103,16 @@ function EventHighLight({ classes }) {
...
+					<Button variant="outlined"
+						classes={{ root: classes.button }}
+					>
```
Comment (GalenWong) [inline src/components/EventHighlight/EventHighlight.js:111]: same as above!

File: src/components/EventHighlight/EventHighlight.js [hunk 3]
```diff
@@ -93,6 +103,16 @@ function EventHighLight({ classes }) {
...
+						{event.button}
+						<LaunchIcon classes={{ root: classes.icon }}/>
```
Comment (GalenWong) [inline src/components/EventHighlight/EventHighlight.js:113]:
```suggestion
						<LaunchIcon classes={{ root: classes.icon }} />
```

We always keep a space before `/` for a self-enclosing tag.

**Comment by TimothyGu** [comment]
Does this fix either #69 or #70? If so you should add this information to the commit messages and pull request descriptions.

**Comment by TimothyGu** [comment]
@jodymlin Next time when you merge a pull request, could you make sure the line

```
Fixes: #70 
```

is in the commit message of the merge commit?

**Comment by jodymlin** [comment]
> @jodymlin Next time when you merge a pull request, could you make sure the line
> 
> ```
> Fixes: #70 
> ```
> 
> is in the commit message of the merge commit?

Whoops sorry my bad

**Pull request merged**